pySDC/projects/Resilience/advection.py

```diff
@@ -40,7 +40,7 @@ def run_advection(
     sweeper_params['num_nodes'] = 3
     sweeper_params['QI'] = 'IE'
 
-    problem_params = {'freq': 2, 'nvars': 2**9, 'c': 1.0, 'type': 'backward', 'order': 5, 'bc': 'periodic'}
+    problem_params = {'freq': 2, 'nvars': 2**9, 'c': 1.0, 'order': 5, 'bc': 'periodic'}
```

Comment:
why did you remove the type here? Did you mean to replace it with stencil_type or did I do sth stupid?

Reply:
Haem ... was maybe due to standard tests that were done with `stencil_type="center"`, even if you could put `type`: `backward` in the previous version, which was simply ignored. I may have done a replacement on all files of this ...
But I'm not so sure : do you want `backward` stencil here specifically ?